packages/currency/src/currencyManager.ts

```diff
@@ -228,28 +228,53 @@ export class CurrencyManager<TMeta = unknown> implements ICurrencyManager<TMeta>
    * Throws if the currency is an ISO4217 currency.
    */
   static validateAddress(address: string, currency: CurrencyInput): boolean {
```

Comment:
regarding the naming what do you think about:
validateAddress into validateCryptoCurrencyAddress, as it validates tokens or native tokens currencies. (no fiat)
validateAddressWithNetworkAndType into validateCurrencyAddress